securedrop_client/gui/auth/__init__.py

```diff
@@ -1,7 +1,7 @@
 """
 Widgets related to authentication.
 
-Copyright (C) 2018  The Freedom of the Press Foundation.
+Copyright (C) 2021  The Freedom of the Press Foundation.
```

Comment:
Thanks for updating the copyright date! We'll want to do this throughout in the near future, but certainly not worth blocking on in the meantime.